agents/node/vcxagent-core/readme.md

```diff
@@ -0,0 +1,15 @@
+# VCX Agent Core
+VCX Agent Core is implementation of simple agent with persistent file storage. It's based on
+[NodeJS Wrapper](../../../wrappers/node) for `aries-vcx` library.
+
+# Try it
+1. First step is to [compile](../../../libvcx) `aries-vcx` library and make it available on your system.
+2. You need to have `aries-vcx` compatible agency ready. See more [info](../../../README.md).
+3. Run `test:threaded:integration`. This will run sample scenario where:Alice and Faber
+    - establish Aries connection, 
+    - Faber issues credential to Alice
+    - Faber requests Alice to prove certain information using the credential. 
+    
+# Note 
+You can also have look at [vcxagent-cli](../vcxagent-cli) - simple CLI Aries agent based 
```

Comment:
"And did we mention how simple it is?" :) Not sure it is necessary keep mentioning it...

Reply:
Yeah the meaning of the "simple" is rather "it does not have many features" or "this is not meant to built on top of for production". But yeah you are right, it's sufficient to say it's using file storage and everyone should be clear :-)